### PR TITLE
Rootlesskit v1.1.1

### PR DESCRIPTION
**Pull request by ghost**
**- What I did**

Update `rootlesskit` to `v1.1.1`.

https://github.com/rootless-containers/rootlesskit/compare/v1.1.0...v1.1.1

`v1.1.1` adds the following fix: https://github.com/rootless-containers/rootlesskit/pull/369
This fix makes rootless Docker work when both `getsubuids "$USER"` and `getsubuids "$(id -u)"` return the same subuid ranges (same for gids).

**- How I did it**

* Change the referenced `rootlesskit` version/revision in `vendor.mod` and `rootlesskit.install`
* Update the vendored packages

**- How to verify it**

* Setup nss such that `getsubuids "$USER"` and `getsubuids "$(id -u)"` return the same subuid ranges
* Install rootless docker
* Run any container

### CODE REVIEW

**Comment by jg-public** [comment]
@AkihiroSuda fyi

**Review by AkihiroSuda — APPROVED** [review]

File: vendor.mod [hunk 1]
```diff
@@ -80,8 +80,8 @@ require (
 	github.com/pelletier/go-toml v1.9.5
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
-	github.com/rootless-containers/rootlesskit v1.1.0
-	github.com/sirupsen/logrus v1.9.0
+	github.com/rootless-containers/rootlesskit v1.1.1
+	github.com/sirupsen/logrus v1.9.2
```
Comment (thaJeztah) [inline vendor.mod:84]: We should update this one to v1.9.3, as v1.9.2 was borked. https://github.com/sirupsen/logrus/releases/tag/v1.9.3

Comment (jg-public) [reply vendor.mod]: solved via https://github.com/moby/moby/pull/45706/commits/57889cebdfee1bd746a59cbf640e0426f24abeed

**Review by neersighted — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**